### PR TITLE
Fix fixture dev deps

### PR DESCRIPTION
**Pull request by askoufis**
As mentioned [here](https://github.com/seek-oss/sku/pull/771#discussion_r1168056948), fixture dev deps weren't really correct. This PR addresses that by:

- Moving `sku` to a dev dep for all fixtures
- Moving/adding `dedent` as a dev dep to relevant fixtures
- Adding some types packages to dev deps

I also added `/* html */` comments to template strings in fixtures so IDEs show correct syntax highlighting.

### CODE REVIEW

File: fixtures/assertion-removal/src/render.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ const skuRender: Render = {
       </SkuProvider>,
     ),
 
-  renderDocument: ({ app, headTags, bodyTags }) => dedent`
+  renderDocument: ({ app, headTags, bodyTags }) => dedent/* html */ `
```
Comment (mrm007) [inline fixtures/assertion-removal/src/render.tsx:16]: Pro tip: if you alias `dedent` to `html` Prettier will format the markup inside the string tag, [like this](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEALGBbAG0wF5MATOC2AbgB0p7IoNMAjAgQ2xJ3wIHoAVL0KZB-AAb0APAEIAIgHkAwgBUAmgAUAoiIIA+epmOZpuQoagmTZuBzKXrmWjBc28cGB0xhsHAE5onsS0IACqqgBiALQAHKGOTsbSMACWMARw+gCyAJ6YAIIA7nBoEB6Ymv4QAFZwYDDS-GkZWUZJph5emFAcHiEgAG6pcEUADhD+riA+0PCwA0WpZDDYxBTDYHDRSyvYADSYqVDpqRwE0Whg53DEAIwJ7UkAJMDYdmSqHADmaAC+TXe9kS1mkrAgZFyIKc0jIqUGRzIAw4YzGCVeKLGAP4cMG0Osr3BkK+vz+Txs-CJUPJgL4lgk1BA+xAEDGaXQyFAAWqRU0AQQaGQIHORQ4uUFzNY-g4YAA1p4AMp9OAAGWOcGQADNzkFJdK5YqxjLjt9kDB-ABXODMuB4ViUChkFUcKDfC0-OCRSZ4DgwNKuoUcC0wCBMkDmAgAdWw6VKRq2CoFp2GMFyQrAaAlIGOQSmVR+Pq1OutIBqaAAHgqTZkAIoWiDwIsEXUgI2BOD+IWsDj2ghhsb+Y4wSPLVbIWIABmZA4gQUj0rGQoHpQ7gw1zIAjvX4FVWYKUBw0NEoHBKJQw-44FvUpf899C0htc2S0E8KkzZaX9W4HWGxrH8WzJeKwI57MgABMQHSqkBAmso5QPiApQAKxhhaQRfKw+5Pi2gxWgAklAVAwAqYCDmyBREQqqaZE2QR-H8QA).

Comment (askoufis) [reply fixtures/assertion-removal/src/render.tsx:16]: That's neat.

File: fixtures/braid-design-system/src/App.test.tsx [hunk 2]
```diff
@@ -15,8 +15,8 @@ describe('braid-design-system', () => {
     ).toMatchInlineSnapshot(`
       "<style type="text/css">
                   html,body{margin:0;padding:0;background:#F7F8FB}
-                  html.sprinkles_darkMode__in90cbz,html.sprinkles_darkMode__in90cbz body{color-scheme:dark;background:#0E131B}
-                </style><div class="apacTheme_default__eah7240 typography_lightModeTone_light__myxsed18 typography_darkModeTone_dark__myxsed1b"><div class="reset_base__pv0jxi0 sprinkles_paddingTop_large_mobile__in90cb6q App_vanillaBox__inn18b0"></div></div>"
+                  html.sprinkles_darkMode__1xl4lgvz,html.sprinkles_darkMode__1xl4lgvz body{color-scheme:dark;background:#0E131B}
```
Comment (mrm007) [inline fixtures/braid-design-system/src/App.test.tsx:18]: Weird, why did these hashes change?

Comment (askoufis) [reply fixtures/braid-design-system/src/App.test.tsx:18]: Maybe this? https://github.com/seek-oss/sku/pull/772/files#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR6406
Honestly not too sure what causes the hash to change half the time.

Comment (mrm007) [reply fixtures/braid-design-system/src/App.test.tsx:18]: There's a Braid above and another one below the one you linked 

braids galore

Comment (mrm007) [reply fixtures/braid-design-system/src/App.test.tsx:18]: > Honestly not too sure what causes the hash to change half the time.

Vanilla Extract computes the hash from the file scope. The file scope contains the file path and when the location of `braid-design-system` changes, so does the hash.

**Review by mrm007 — APPROVED** [review]

**Pull request merged**